src/tests/unit/tests/electron/views/screenshot/screenshot-view.test.tsx

```diff
@@ -5,10 +5,22 @@ import { shallow } from 'enzyme';
 import * as React from 'react';
 
 describe('ScreenshotView', () => {
-    it('renders', () => {
-        const props: ScreenshotViewProps = {};
-        const wrapper = shallow(<ScreenshotView {...props} />);
+    describe('renders', () => {
+        it('when passed a value for screenshotData', () => {
+            const props: ScreenshotViewProps = {
+                screenshotData: { base64PngData: 'string' },
+            };
+            const wrapper = shallow(<ScreenshotView {...props} />);
 
-        expect(wrapper.getElement()).toMatchSnapshot();
+            expect(wrapper.getElement()).toMatchSnapshot();
+        });
+
+        it('when passed an empty value for screenshotData', () => {
+            const props = {} as ScreenshotViewProps;
```

Comment:
I think you should cover `props.screenshotData` being empty. It may also be worthy to check for `null` and `undefined` values, as it can be done using `it.each`